src/pages/ProductPage/ProdDetail/ProdDetail.jsx

```diff
@@ -0,0 +1,23 @@
+import "./ProdDetail.scss";
+
+export default function ProdDetail({
+  company,
+  model,
+  description,
+  originalPrice,
+  price,
+  discount,
+}) {
+  return (
+    <div className="prodDetail">
+      <h2>{company}</h2>
+      <h1>{model}</h1>

```

Comment:
Same here, let's always add a class:

```suggestion
      <h1 className="product-detail__title">{model}</h1>
```